src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -169,3 +169,49 @@ bool Terminal::SetColorTableEntry(const size_t tableIndex, const DWORD dwColor)
     _buffer->GetRenderTarget().TriggerRedrawAll();
     return true;
 }
+
+// Method Description:
+// - Sets the cursor style to the given style.
+// Arguments:
+// - cursorStyle: the style to be set for the cursor
+// Return Value:
+// - true iff we successfully set the cursor style
+bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
+{
+    CursorType finalCursorType = CursorType::Legacy;
+    bool shouldBlink = false;

```

Comment:
Per the ISO C++ Guidelines:
[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

This also applies to the line above.

For this line, combined with naming pattern consistency, consider:
`bool fShouldBlink{};`